frontend/src/js/components/PageViewer/VirtualScroll.tsx

```diff
@@ -101,27 +103,39 @@ export const VirtualScroll: FC<VirtualScrollProps> = ({
     });
   }, [pageHeight, totalPages, debouncedSetPageRange]);
 
-  useEffect(() => {
-    // Inform the parent component of the new middle page
-    // This allows it to do useful things such as have a sensible "next" page
-    // to go to for the find hits
-    onMiddlePageChange(pageRange.middle);
-  }, [pageRange.middle, onMiddlePageChange]);
-
   const throttledSetPageRangeFromScrollPosition = useMemo(() => throttle(setPageRangeFromScrollPosition, 75), [setPageRangeFromScrollPosition]);
 
+  // TODO: try just useEffect
   useLayoutEffect(() => {
-    if (viewport?.current && jumpToPage) {
+    if (viewport?.current) {
       const v = viewport.current;
-      const scrollTo = (jumpToPage - 1) * pageHeight;
-      v.scrollTop = scrollTo;
+      if (jumpToPage) {
+        // TODO: page jump should go to middle?
+        const scrollTo = (jumpToPage - 1) * pageHeight;
+        v.scrollTop = scrollTo;
+      }
+      if (jumpToScrollPosition) {
```

Comment:
didn't use `jumpToScrollPosition` in the end but have left it in

Reply:
Shall we get rid? I think just viewing pages at a time is probably ok?